### DISCUSSION TITLE
Post-Judging QA

### DISCUSSION THREAD

**Discussion by liveactionllama** [code-423n4/2024-08-superposition-findings#174] [Announcements]
1. Please post feedback about _specific findings_ on the relevant issue in this repo.
2. Post a comment in this discussion linking to all findings you've commented on, along with any general feedback for the judge.

The judge for this contest is @alex-ppg.

**Both the Validation repo and the Findings repo will be open to wardens with the SR role, for the purposes of post-judging QA.**
- All post-judging QA requests must be posted in this Github Discussion in the findings repo.
- QA and Gas reports closed by Validators (i.e. not added to the findings repo) are NOT eligible.

## Reminders

- Please keep your comments as fact-based as possible.
- C4 judges have final authority in determining validity and severity.
- Please link to specific issues and don't try to start discussions here.
- Please familiarize yourselves with our [Good Citizen Policy](https://docs.code4rena.com/roles/wardens/submission-guidelines#good-citizenship-is-a-requirement-for-compensation).

Thank you!

**Comment by adeolu98** [comment]
hi @alex-ppg, thanks for judging. i left a comment on #146, please check.

**Comment by rodiontr** [comment]
Hi @alex-ppg ! Thanks for your quick judging! I left many comments on many issues so sorry for that but would really appreciate if you take a look:

From the validation repo:
https://github.com/code-423n4/2024-08-superposition-validation/issues/140
https://github.com/code-423n4/2024-08-superposition-validation/issues/166
https://github.com/code-423n4/2024-08-superposition-validation/issues/204
https://github.com/code-423n4/2024-08-superposition-validation/issues/160

https://github.com/code-423n4/2024-08-superposition-validation/issues/169
(the following issues are associated with this finding but are different as one issue does not fix the other one:
https://github.com/code-423n4/2024-08-superposition-validation/issues/179
https://github.com/code-423n4/2024-08-superposition-validation/issues/171)


From the findings repo:

#108 
#54 
#117
#116
#120 
#130

**Reply by alex-ppg** [reply]
Hey @rodiontr, you have raised concerns for 13 separate exhibits. I have evaluated each one, and have concluded that all have no merit given that they rely on misconceptions, are invalid, outline documentation mismatches of little importance, and so on. 

I would strongly advise you to specify at most 3 exhibits you have an issue with to receive a detailed response during the PJQA process.

**Reply by rodiontr** [reply]
> Hey @rodiontr, you have raised concerns for 13 separate exhibits. I have evaluated each one, and have concluded that all have no merit given that they rely on misconceptions, are invalid, outline documentation mismatches of little importance, and so on.
> 
> I would strongly advise you to specify at most 3 exhibits you have an issue with to receive a detailed response during the PJQA process.

got it, thanks for the feedback anyway

**Comment by Slavchew** [comment]
Hey @alex-ppg, left a comment on #68

**Reply by Slavchew** [reply]
Hey @alex-ppg I provided the real-world example proving that this issue is valid.

**Reply by Slavchew** [reply]
@alex-ppg can you please check #68 again. Why is the issue closed?

**Comment by blckhv** [comment]
Hey @alex-ppg, 

I left some comments on the following issues from the validation repo:
- https://github.com/code-423n4/2024-08-superposition-validation/issues/254
- https://github.com/code-423n4/2024-08-superposition-validation/issues/239
- https://github.com/code-423n4/2024-08-superposition-validation/issues/250
- https://github.com/code-423n4/2024-08-superposition-validation/issues/216

Thanks!

**Reply by blckhv** [reply]
Answered to https://github.com/code-423n4/2024-08-superposition-validation/issues/250#issuecomment-2396696157

Thanks!

**Comment by cryptostaker2** [comment]
Hi @alex-ppg , thanks for judging!

Left a comment in validation [108](https://github.com/code-423n4/2024-08-superposition-validation/issues/108).

Thanks!

**Comment by br0nz3p1ck4x3** [comment]
Hi @alex-ppg,

Left a comment over at https://github.com/code-423n4/2024-08-superposition-validation/issues/24

Please let us know if you need any additional input.

Thanks!

**Comment by devival** [comment]
Hello @alex-ppg,
Left a comment for 
- https://github.com/code-423n4/2024-08-superposition-validation/issues/217

Thank you!

**Comment by 0xSilvermist** [comment]
Hi @alex-ppg, 
I left a comment on #62.

**Comment by 0xTester** [comment]
Hi @alex-ppg, I left a comment on #53 
Thank you!

**Comment by ZanyBonzy** [comment]
Hi @alex-ppg , thanks for the work. Left comments on #138 and [here](https://github.com/code-423n4/2024-08-superposition-validation/issues/66). Would love to hear your thoughts.

**Comment by Ys-Prakash** [comment]
Hi @alex-ppg 

I have left comments on #77, #48, #167

Thank you

**Reply by Ys-Prakash** [reply]
Hi @alex-ppg 

I replied to #77

Thank you

**Reply by Ys-Prakash** [reply]
Hi @alex-ppg

I replied to #167

Thank you

**Comment by DadeKuma** [comment]
Hello @alex-ppg,

Here are my comments:

Findings:
https://github.com/code-423n4/2024-08-superposition-findings/issues/61
https://github.com/code-423n4/2024-08-superposition-findings/issues/48

Validation:
https://github.com/code-423n4/2024-08-superposition-validation/issues/51

I think most downgraded QA were miscategorized as `grade-c unsatisfactory` for Wardens with at least a `grade-b` QA report.

For example, my QA report has a `grade-A`, but this downgraded issue: https://github.com/code-423n4/2024-08-superposition-findings/issues/139 has `grade-C`, and thus it's `unsatisfactory`.

This will unfairly reduce the accuracy score, so it would be better to fix these labels as they should have the same grade as the original QA report (not only mine, but every downgraded HM for other Wardens).

**Comment by thebrittfactor** [comment]
Please note: PJ QA has now closed. No additional comments should be added unless requested by the judge.
